dashboards-observability/public/components/visualizations/charts/histogram/histogram_type.ts

```diff
@@ -75,6 +76,39 @@ export const createHistogramVisDefinition = (params = {}) => ({
             mapTo: 'colorTheme',
             schemas: [],
           },
+          {
+            id: 'tooltip_options',
+            name: 'Tooltip options',
+            editor: ConfigTooltip,
+            mapTo: 'tooltipOptions',
+            schemas: [
+              {
+                name: 'Tooltip mode',
+                component: null,
+                mapTo: 'tooltipMode',
+                props: {
+                  options: [
+                    { name: 'Show', id: 'show' },
+                    { name: 'Hidden', id: 'hidden' },
+                  ],
+                  defaultSelections: [{ name: 'Show', id: 'show' }],
+                },
+              },
+              {
+                name: 'Tooltip text',
+                component: null,
+                mapTo: 'tooltipText',
+                props: {
+                  options: [
+                    { name: 'All', id: 'all' },
+                    { name: 'Dimension', id: 'x' },
+                    { name: 'Metrics', id: 'y' },
+                  ],
+                  defaultSelections: [{ name: 'All', id: 'all' }],
+                },
+              },
+            ],
+          },
```

Comment:
This code looks like repetitive most of the places. Can't we create some common file and get this as a return value of the function. Some variables like `editor: ConfigTooltip` can be provided as function arguments.